server/apps/main/templates/main/signup.html

```diff
@@ -115,7 +115,7 @@
                 <input type="radio" name="inlineRadioOptions" id="inlineRadio4" value="option4">
             </div>
             <div class="submit">
-                <button type="button" class="btn btn-primary mb1 bg-greenyellow" form="form-check">CONTINUE</button>
+                <a href="/server/apps/main/templates/main/signup1.html"><button type="button" class="btn btn-primary mb1 bg-greenyellow" form="form-check">CONTINUE</button></a>
```

Comment:
Hey @mridubhatnagar , so for this href, you don't provide the absolute path to the html file on the server. Rather, you provide the _URL_ path/pattern that directs to the `signup1.html` page. 

For example, the URL pattern that directs users, according to the `urls.py` that you've modified is `signup1`. The django app, which is what we're developing, is called `main`. 

So the `href` link would now become `/main/signup1`. 

Currently, if I try to navigate to the second page, I get this 404 error .
![CleanShot 2021-07-08 at 07 45 07](https://user-images.githubusercontent.com/1872836/124916268-7aaff980-dfc0-11eb-855b-625feb7ec325.jpg)

Reply:
Hey @jhlink 

Yes, makes sense. 

So, I tried replacing the link to /main/signup1/ and tested the implementation on my local. The redirection to the next page doesn't work. I get an error saying 

```
Cannot GET /main/signup1
```

I then modified the URL to Django template-based URL that we earlier discussed. 

```
<div class="submit">
                <a href="{% url 'main:signup_frame4_test' %}"><button type="button" class="btn btn-primary mb1 bg-greenyellow" form="form-check">CONTINUE</button></a>
</div>
```
On using the above URL I get a bad request error.

Reply:
Awesome. This works great!!